### PR TITLE
Added filter-modal, sidebar & toast

### PR DESCRIPTION
**Pull request by AbhishekkGautam**
### Filter Modal for Mobile Screen.
- Sort 
mobile (3)

- Filters
mobile (4)

### Sidebar Navigation
mobile (5)

### Toast Alerts
- Log in ( welcome user )

Screen Shot 2022-03-25 at 4 07 25 AM 1

- Log out 

Screen Shot 2022-03-25 at 4 08 18 AM

- wishlist/cart

Screen Shot 2022-03-25 at 4 06 21 AM

Screen Shot 2022-03-25 at 4 05 07 AM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,13 +4,28 @@ import Mockman from "mockman-js";
...
+          // Define default options
+          className: "",
+          style: {
+            minWidth: "280px",
+          },
+          success: {
+            duration: 2000,
+          },
+        }}
+      />
```
Comment (BelideNikhil) [inline src/App.js]: You can extract this into a component and then use it here so that the App.js looks clean!
Same with Routes!

**Review by BelideNikhil — APPROVED** [review]
Great work man!
The code is clean.
The sidebar and filter animations look good as well.
Add guest login and do form validation as well
GTG!

File: src/components/ExploreCategories/ExploreCategories.js [hunk 2]
```diff
@@ -23,7 +27,17 @@ export const ExploreCategories = () => {
...
               categories.map((category, id) => {
                 return (
-                  <div className="category-card" key={id}>
+                  <div
+                    className="category-card"
+                    key={id}
+                    onClick={() => {
+                      dispatch({
+                        type: "FILTER_BY_CATEGORIES",
+                        payload: category,
```
Comment (msdeshmukh009) [inline src/components/ExploreCategories/ExploreCategories.js:37]: Maybe you should clear the existing filters first and then dispatch this action.

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking clean and readable. Good to merge.

File: src/App.js (hunk 1 above)
Comment (AbhishekkGautam) [reply src/App.js]: will refactor it. thank you

File: src/components/ExploreCategories/ExploreCategories.js (hunk 2 above)
Comment (AbhishekkGautam) [reply src/components/ExploreCategories/ExploreCategories.js:37]: done. thank you

**Pull request merged**